src/main/java/com/epam/ta/reportportal/core/widget/content/loader/LaunchesTableContentLoader.java

```diff
@@ -22,22 +22,51 @@
 package com.epam.ta.reportportal.core.widget.content.loader;
 
 import com.epam.ta.reportportal.commons.querygen.Filter;
+import com.epam.ta.reportportal.commons.validation.BusinessRule;
 import com.epam.ta.reportportal.core.widget.content.LoadContentStrategy;
+import com.epam.ta.reportportal.dao.WidgetContentRepository;
+import com.epam.ta.reportportal.entity.widget.ContentField;
 import com.epam.ta.reportportal.entity.widget.WidgetOption;
+import com.epam.ta.reportportal.entity.widget.content.LaunchStatisticsContent;
+import com.epam.ta.reportportal.ws.model.ErrorType;
+import org.apache.commons.collections.MapUtils;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import static com.epam.ta.reportportal.commons.Predicates.equalTo;
+import static com.epam.ta.reportportal.core.widget.content.WidgetContentUtils.GROUP_CONTENT_FIELDS;
+import static java.util.Collections.singletonMap;
+
 /**
  * @author Pavel Bortnik
  */
 @Service
 public class LaunchesTableContentLoader implements LoadContentStrategy {
 
+	@Autowired
+	private WidgetContentRepository widgetContentRepository;
+
 	@Override
-	public Map<String, ?> loadContent(List<String> contentFields, Filter filters, Set<WidgetOption> widgetOptions, int limit) {
-		return null;
+	public Map<String, ?> loadContent(Set<ContentField> contentFields, Filter filter, Set<WidgetOption> widgetOptions, int limit) {
+
+		Map<String, List<String>> fields = GROUP_CONTENT_FIELDS.apply(contentFields);
+		validateContentFields(fields);
+
+		List<LaunchStatisticsContent> result = widgetContentRepository.launchesTableStatistics(filter, fields, limit);
+		return singletonMap(RESULT, result);
+	}
+
+	/**
+	 * Validate provided content fields.
+	 *
+	 * @param contentFields Map of provided content.
+	 */
+	private void validateContentFields(Map<String, List<String>> contentFields) {
```

Comment:
let's add some more validation on required content fields keys. for example, if we know that content fields should content either 'execution' or 'defects' key, we probably need to validate that.

Reply:
and provided more detailed docs on restrictions